src/OverlayTrigger.js

```diff
@@ -31,7 +31,7 @@ class OverlayTrigger extends Overlay {
 
   showOverlay (e) {
     e.preventDefault();
-    $(`#${this.overlayID}`).modal(this.props.modalOptions).modal('open');
+    $(`#${this.overlayID}`).modal('open', this.props.modalOptions);
```

Comment:
I'm not sure about this, it seems to work fine as it currently is, does it not work for you?

Reply:
he might mean there is not default value for `props.modalOptions`, and it doesn't work when the user didn't provide one?

Reply:
ah I see, then maybe we could add `defaultProps: { modalOptions: {} }`?

Reply:
the way he proposed looks ok, why do we chain call `.modal`?

Reply:
first we initialise and then we call it, but sure, I have no objection with the proposed solution. I just didn't know there was an initial issue. But now I see :)